### PR TITLE
[skills]update scripts

### LINKED ISSUE #78: [Skills] deployment script update
#### What project is affected?
All skills
#### What happens?
1. Align deployment script to skill template

2. change the  "group deployment" to "deployment group"
image

### PR DESCRIPTION
**Pull request by xhr0804**
### Purpose
*What is the context of this pull request? Why is it being done?*
close #78 
related to #124
### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/experimental/newsskill/Deployment/Resources/template.json [hunk 1]
```diff
@@ -94,14 +102,6 @@
     "luisAuthoringLocation": {
       "type": "string"
     },
-    "bingServiceName": {
```
Comment (xieofxie) [inline skills/csharp/experimental/newsskill/Deployment/Resources/template.json]: keep these?

File: skills/csharp/experimental/newsskill/Deployment/Resources/template.json [hunk 2]
```diff
@@ -320,16 +335,9 @@
         "region": "[parameters('luisRuntimeLocation')]",
         "key": "[listKeys(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisRuntimeName')),'2016-02-01-preview').key1]",
         "authoringRegion": "[parameters('luisAuthoringLocation')]",
-        "authoringKey": "[if(parameters('useLuisAuthoring'), listKeys(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisAuthoringName')),'2016-02-01-preview').key1, '')]"
+        "authoringKey": "[if(parameters('useLuisAuthoring'), listKeys(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisAuthoringName')),'2016-02-01-preview').key1, '')]",
+        "endpoint": "[if(parameters('useLuisAuthoring'), reference(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisAuthoringName')),'2016-02-01-preview').endpoint, '')]"
       }
-    },
-    "BingNewsKey": {
```
Comment (xieofxie) [inline skills/csharp/experimental/newsskill/Deployment/Resources/template.json]: keep?

File: skills/csharp/pointofinterestskill/Deployment/Resources/template.json [hunk 3]
```diff
@@ -94,10 +102,6 @@
     "luisAuthoringLocation": {
       "type": "string"
     },
-    "azureMapsServiceName": {
```
Comment (xieofxie) [inline skills/csharp/pointofinterestskill/Deployment/Resources/template.json]: keep?

File: skills/csharp/pointofinterestskill/Deployment/Resources/template.json [hunk 4]
```diff
@@ -276,16 +295,6 @@
         "[resourceId('Microsoft.CognitiveServices/accounts', parameters('luisRuntimeName'))]"
       ],
       "condition": "[parameters('useLuisAuthoring')]"
-    },
-    {
-      "type": "Microsoft.Maps/accounts",
```
Comment (xieofxie) [inline skills/csharp/pointofinterestskill/Deployment/Resources/template.json]: keep?

File: skills/csharp/pointofinterestskill/Deployment/Resources/template.json [hunk 5]
```diff
@@ -326,12 +335,9 @@
         "region": "[parameters('luisRuntimeLocation')]",
         "key": "[listKeys(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisRuntimeName')),'2016-02-01-preview').key1]",
         "authoringRegion": "[parameters('luisAuthoringLocation')]",
-        "authoringKey": "[if(parameters('useLuisAuthoring'), listKeys(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisAuthoringName')),'2016-02-01-preview').key1, '')]"
+        "authoringKey": "[if(parameters('useLuisAuthoring'), listKeys(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisAuthoringName')),'2016-02-01-preview').key1, '')]",
+        "endpoint": "[if(parameters('useLuisAuthoring'), reference(resourceId('Microsoft.CognitiveServices/accounts', parameters('luisAuthoringName')),'2016-02-01-preview').endpoint, '')]"
       }
-    },
-    "azureMapsKey": {
```
Comment (xieofxie) [inline skills/csharp/pointofinterestskill/Deployment/Resources/template.json]: keep?

File: skills/csharp/experimental/newsskill/Deployment/Resources/template.json (hunk 2 above)
Comment (xhr0804) [reply skills/csharp/experimental/newsskill/Deployment/Resources/template.json]: fixed

File: skills/csharp/experimental/newsskill/Deployment/Resources/template.json (hunk 1 above)
Comment (xhr0804) [reply skills/csharp/experimental/newsskill/Deployment/Resources/template.json]: fixed

File: skills/csharp/emailskill/Deployment/Resources/template.json [hunk 6]
```diff
@@ -107,13 +115,12 @@
...
     "storageAccountName": "[toLower(take(replace(replace(parameters('storageAccountName'), '-', ''), '_', ''), 24))]",
-    "cosmosDbAccountName": "[toLower(take(replace(parameters('cosmosDbName'), '_', ''), 31))]",
-    "botEndpoint": "[concat('https://', toLower(variables('botWebAppName')), '.azurewebsites.net/api/messages')]"
+    "cosmosDbAccountName": "[toLower(take(replace(replace(parameters('cosmosDbName'), '-', ''), '_', ''), 31))]"
   },
   "resources": [
     {
       "apiVersion": "2018-02-01",
-      "name": "df123cff-9cd6-4290-a7a7-113afe2363f7",
+      "name": "99ea37e6-a3e6-4102-a249-71c880607386",
```
Comment (xieofxie) [inline skills/csharp/emailskill/Deployment/Resources/template.json]: what's the meaning of this name? They are all different before

Comment (xhr0804) [reply skills/csharp/emailskill/Deployment/Resources/template.json]: looks like some skills have the same name with the sample while others not. Anyway, Kept it as original

**Pull request merged**